app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -0,0 +1,25 @@
+module AdminOnly
+  
+  class MemberAppWaitingReason < ApplicationRecord
+
+    validates_presence_of :name_sv
```

Comment:
Related to this is editing a `reason`.    If the reason has MA's associated with it, then the admin should be aware of that so that any edit to the `reason` does not actually negate the original intent of the `reason`.   

That is, if 5 MA's refer to `reason` "Need training certification document", the reason is edited to say "Need grooming training certificate", then the 5 MA's now have invalid information.

Perhaps the form could 1) indicated how many MA's (regardless of MA state) actually use this reason, and 2) provide some help text that reminds the admin not to alter the _intent_ of the `reason`.